app/views/images/new.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="container-fluid">
   <%= simple_form_for @image do |f| %>
     <%= f.input :imageurl, as: :url, label: "Image URL" %>
+    <%= f.input :tag_list, as: :string, label: "Tags" %>
```

Comment:
If you submit the form with an invalid URL and comma separated tags, are the tags still comma separated after you got back an error?

Reply:
Interestingly, no.  It seems that when the `tag_list` array is rendered into the input field, it is rendered with only spaces delineating the entries.  Will address.